src/stactools/ecmwf_forecast/stac.py

```diff
@@ -357,20 +346,25 @@ def _create_item_from_parts(parts: list[Parts], split_by_step=False) -> Item:
     item.properties["ecmwf:stream"] = part.stream
     item.properties["ecmwf:type"] = part.type
     item.properties["ecmwf:reference_datetime"] = (
-        part.reference_datetime.isoformat() + "Z"
-    )
+        part.reference_datetime.isoformat() + "Z")
     item.properties["ecmwf:forecast_datetime"] = (
-        part.forecast_datetime.isoformat() + "Z"
-    )
+        part.forecast_datetime.isoformat() + "Z")
+
+    fs = fsspec.filesystem('')
+    fs.clear_instance_cache()
+
+    #remove this once this has been tested for all 7 collections
+    if ((part.stream=='wave') & (part.type=='fc')):
+        item.properties["kerchunk_indices"] = khf.get_kerchunk_indices(part)
```

Comment:
We can keep this focused on Kerhcunk.

As a followup, we can add the datacube stuff from `xstac`.

Reply:
```suggestion
        item.properties["kerchunk:indices"] = khf.get_kerchunk_indices(part)
```

Most STAC extensions namespace their values under a `:` prefix. Let's do the same here.